transcription/transcription.go

```diff
@@ -29,3 +32,25 @@ func msgHeaders(from string, to []string, subject string) string {
 	msgHeaders := []string{fromHeader, toHeader, subjectHeader}
 	return strings.Join(msgHeaders, "\r\n")
 }
+
+// Transcribe a given file using a Sphinx jar. The fileName should be in
+// "name.wav" format and the jarName should be in "name.jar" format.
+func startTranscription(fileName string, jarName string) error {
```

Comment:
@yoninachmany can you link or quote from relevant Go documetnation material (e.g. effective Go) when you give comments on Go syntax/practices